app/2.0/start/first-element/step-2.md

```diff
@@ -62,6 +62,7 @@ Starting code—local DOM template { .caption }
     <!-- shadow DOM goes here -->
     <span>Not much here yet.</span>
   </template>
+</dom-module>
```

Comment:
This is just showing the contents of the existing files, so it doesn't need to be copy-pastable. (The closing `<dom-module>` tag follows the `<script>`, shown in the next listing.)